data/script_cmd_table.inc

```diff
@@ -1,232 +1,249 @@
+	.ifndef SCRIPT_CMD_TABLE_ENTRY_MACRO
+	.set SCRIPT_CMD_TABLE_ENTRY_MACRO, 1
+	.macro script_cmd_table_entry constant:req value:req
+	.ifndef ALLOCATE_SCRIPT_CMD_TABLE
+	enum \constant
+	.else
+	.4byte \value
+	.endif
```

Comment:
I think `.if`/`.ifne` would be clearer than `.ifdef`/`.ifndef` (and are necessary with an explicit `.set ALLOCATION_SCRIPT_CMD_TABLE, 0`), e.g.:
```suggestion
	.if ALLOCATE_SCRIPT_CMD_TABLE
	.4byte \value
	.else
	enum \constant
	.endif
```